### PR TITLE
Obmin 415 main animation block

### PR DESCRIPTION
**Pull request by Kyslytsia**

### CODE REVIEW

File: src/components/site-purpose/animation.ts [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  0% {
+    left: 0;
+  }
+
+  27% {
+    left: 0;
+  }
+  
+  33% {
+    opacity: 1;
```
Comment (YakovenkoIhor) [inline src/components/site-purpose/animation.ts:59]: swap left and oppacity

File: src/components/site-purpose/animation.ts [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+export const ColunmAnimation3 = keyframes`
+  0% {
+    left: 200%;
+    opacity: 0;
+  }
+
+  27% {
+    left: 200%;
+    opacity: 1;
+  }
```
Comment (YakovenkoIhor) [inline src/components/site-purpose/animation.ts:130]: indent

File: src/components/site-purpose/featuresSection/featuresClouds/styles.ts [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+      height: 65px;
+      animation: ${Animation.RightMiddleCloud} 15s linear infinite;
+    `}
+  `}
+`;
+
+export const RightBottomCloud = styled.div`
+  ${cloudStyles};
+
+  right: 1%;
```
Comment (YakovenkoIhor) [inline src/components/site-purpose/featuresSection/featuresClouds/styles.ts:229]: swap lines

File: src/components/site-purpose/featuresSection/featuresClouds/styles.ts [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+      animation: ${Animation.RightBottomCloud} 15s linear infinite;
+    `}
+  `}
+`;
+
+export const HappySun = styled.div`
+  position: absolute;
+  top: 10%;
+  left: 10%;
+  height: 83px;
```
Comment (YakovenkoIhor) [inline src/components/site-purpose/featuresSection/featuresClouds/styles.ts:265]: swap lines

**Review by YakovenkoIhor — APPROVED** [review]
checked)

File: src/components/site-purpose/animation.ts [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+  from {
+    opacity: 0;
+    transform: scale(0);
+  }
+
+  to {
+    opacity: 1;
+    transform: scale(1);
+  }
+`;
```
Comment (ASGladchenko) [inline src/components/site-purpose/animation.ts:13]: bad name for for this animation mb fadeOut

**Review by VadimALF — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**